README.md

```diff
@@ -397,12 +397,10 @@ direct_greedy_cpus=f
 - scx_lavd
 
 ```
-❯ scx_lavd --monitor-sched-samples 1
-    stat: ('L'atency-critical, 'R'egular) (performance-'H'ungry, performance-'I'nsensitive) ('B'ig, li'T'tle) ('E'ligigle, 'G'reedy) ('P'reempting, 'N'ot)
-|    428 |  111375 | scx_lavd          | RHBGN |    2 |   -1 |        8703000 |  3000000 |   34812 |       12 |      70 |       20 |       0 |         0 |   1702362 |         0 |         0 |       22 |       84 |      822 |       53 |      1 |
-|    429 |  111375 | scx_lavd          | RHBGP |    2 |    1 |         145411 |  3000000 |    2472 |       51 |      71 |       20 |       0 |      2976 |     95521 |     26907 |     10348 |       71 |       85 |      393 |       26 |      1 |
-|    430 |  111375 | scx_lavd          | RHBGN |    3 |   -1 |          83886 |  3000000 |    1482 |       53 |      72 |       20 |       0 |      5681 |     51129 |     16819 |     21793 |       72 |       85 |     1024 |       67 |      1 |
-|    431 |  105466 | scx_lavd          | RHBEN |    2 |   -1 |              0 |  3000000 |       1 |       71 |      72 |       20 |       0 |      2037 |     38775 |      1404 |      1969 |       60 |       86 |      919 |       50 |      1 |
+❯ cx_lavd --monitor 5
```

Comment:
One typo here: 's' is missing in 'cx_lavd'

Reply:
Fixed! Thanks